packages/sdk/src/cross-chain-messenger.ts

```diff
@@ -1563,9 +1563,17 @@ export class CrossChainMessenger {
     opts?: {
       signer?: Signer
       overrides?: Overrides
-    }
+    },
+    /**
+     * The index of the withdrawal if multiple are made with multicall
```

Comment:
Not sure I will understand what this mean out of context or in a few months, can this be worded differently?

Reply:
Also bump on moving the comment to the proper docstring for these functions

Reply:
Will update in follow up pr since this comment is added in more places than just in this pr

@hamdiallam putting it here will end up in the correct docstring post compilation. TS is smart about putting them in the right spot.  This way of doing it is more idiomatic for TS builds